.eslintrc.js

```diff
@@ -6,7 +6,10 @@ const process = require('process');
 const isRelaxed = !!process.env.SENTRY_ESLINT_RELAXED;
 
 module.exports = {
-  extends: [isRelaxed ? 'sentry-app' : 'sentry-app/strict'],
+  extends: [
```

Comment:
Changes introduced here need to be removed before we can merge

Reply:
Gonna move it the the sentry-eslint-config repo right?